orchestration/dagster_orchestration/hca_orchestration/solids/copy_project/subgraph_hydration.py

```diff
@@ -209,11 +197,18 @@ def fetch_entities(
         bigquery_service: BigQueryService) -> dict[str, list[Row]]:
     result: dict[str, list[Row]] = defaultdict(list[Row])
     for entity_type, entities in entities_by_type.items():
-        fetch_entities_query = f"""
-        SELECT * EXCEPT (datarepo_row_id)
-        FROM {bigquery_project_id}.{snapshot_name}.{entity_type} WHERE {entity_type}_id IN
-        UNNEST(@entity_ids)
-        """
+        if not entity_type.endswith("_file"):
+            fetch_entities_query = f"""
+            SELECT * EXCEPT (datarepo_row_id)
+            FROM `{bigquery_project_id}.{snapshot_name}.{entity_type}` WHERE {entity_type}_id IN
+            UNNEST(@entity_ids)
+            """
+        else:
+            fetch_entities_query = f"""
+            SELECT '/' || JSON_EXTRACT_SCALAR(descriptor, '$.file_id') || '/' || JSON_EXTRACT_SCALAR(descriptor, '$.file_name') as target_path, * EXCEPT (datarepo_row_id)
```

Comment:
I am constructing the desired target path here to avoid having to reconstruct this path downstream during inject_file_ids